Mathlib/Analysis/SpecialFunctions/Pow/Deriv.lean

```diff
@@ -131,6 +131,22 @@ theorem DifferentiableWithinAt.const_cpow (hf : DifferentiableWithinAt ℂ f s x
   (hf.hasFDerivWithinAt.const_cpow h0).differentiableWithinAt
 #align differentiable_within_at.const_cpow DifferentiableWithinAt.const_cpow
 
+theorem DifferentiableOn.cpow (hf : DifferentiableOn ℂ f s) (hg : DifferentiableOn ℂ g s)
+    (h0 : ∀ x ∈ s, f x ∈ slitPlane) : DifferentiableOn ℂ (fun x ↦ f x ^ g x) s :=
```

Comment:
Is this the best spelling for `h0` or should it be something like `s ⊆ f ⁻¹' slitPlane`?

Reply:
It should be `MapsTo f s slitPlane`.